### PR TITLE
13240 - Docs - Working on a full pass through doc/ReferenceManual

### PR DESCRIPTION
**Pull request by Cattlesquat**
(1) Screenshots updated where needed to be accurate for 3.3.3
(2) Articles updated where appropriate for 3.3.3 accuracy
(3) Edit-pass through HTM files - cleaned up the HTML itself and where relevant the text
(4) Additional articles, tables, and illustrations added.

### CODE REVIEW

**Comment by BrentEaston** [comment]
A couple of things

1. The GamePiece page is missing a link to the CalculateProperty page.
2. We really need some sort of TOC and/or Index page where you can go and see all the links to all pages in one place.

**Comment by Cattlesquat** [comment]
For sure! There's actually an Index.htm that as far as I can tell isn't actually even used as a landing page (you can only get there by navigating on the little top bar thing), and it at least goes through the top-level Editor commands, but it isn't actually even linked from the Editor (which instead wants to load something from the website and/or load the PDF) and stuff.  Although this PR I'm trying to keep "pure html", I'm planning a little followon code PR to fix the Editor menu and stuff. 

And I am indeed planning some kind of "Top Level Vassal Module Design Landing Page" that will more easily let you find some of the key areas to explore, etc. And yeah, a TOC is really needed too, yay.

The bottom of GamePiece (where it haphazardly links to about 80% of the traits and not to the others) is actually next on my "list-o-things" -- I've been working my way down through GamePiece as you can probably tell if you boot it, and have just reached that. 

Map.htm is presently nearer to my standard of What A Big-Ass List of Sub-Components Should Look Like.

**Comment by BrentEaston** [comment]
Excellent work. Sounds like you are on top of it.

**Comment by Cattlesquat** [comment]
@uckelman This PR is entirely non-code, and ready to be merged (IMHO at least). I still have ongoing improvements to make, but I can make a new PR for them when this one merges in. It's not quite time for @yanlyub to move all the reference manual directories   (or alternatively, give me a heads up to "get out of the pool").

I will also do a little mini-PR related to this to adjust the contents of the Editor's "Help" menu to be a little more useful.

File: doc/ReferenceManual/GameModule.htm [hunk 1]
```diff
@@ -5,592 +5,764 @@
...
+    <br>
+    </td>
     </tr>
-
+    </tbody>
+  </table>
+  <hr size="2" width="100%">
+  <a href="Toolbar.htm"><h2>Toolbar</h2></a><br>
+
+  <img src="images/Toolbar.png"><br>
```
Comment (uckelman) [inline doc/ReferenceManual/GameModule.htm:74]: Broken image.

Comment (Cattlesquat) [reply doc/ReferenceManual/GameModule.htm:74]: It's a new image, so if you haven't pulled down the full PR it won't find anything.

Comment (Cattlesquat) [reply doc/ReferenceManual/GameModule.htm:74]: fixed

**Pull request merged**